### ISSUE TITLE
runFastQC dies when any part of a project directory contains spaces

### ISSUE THREAD

**Issue by amcdavid** [RGLab/RNASeqPipelineR#1]
In `/shared/silo_researcher/Gottardo_R/gfinak_working/Project - Atreca Single Cell RNASeq/BTK` open R.

```
loadProject('./', 'BTK')
runFastQC(4)
```

I've already tried enquoting the filenames in `run_command` but fastqc seems to not like spaces (or dashes?) in the output directory.  One work-around might be to store the relative paths in getConfig, rather than absolute, since relative to the project root, the file names will be well-behaved. But that might cause it's own share of problems down the line?

**Comment by gfinak** [comment]
Or don't use spaces.
Relative paths would be okay.. until the working directory unexpectedly changes (which is not impossible given that it's run from inside R). 
Otherwise need to handle escaping spaces in an operating system dependent manner.

I prefer option 1) Don't use spaces.

**Comment by amcdavid** [comment]
I agree that don't use spaces is the best option, but the user may not have control of the name of higher-level directories.

In that vein, can you rename the project directory for me?  I don't have permission :-)

**Comment by gfinak** [comment]
In all seriousness, I should have quoted paths, that would handle spaces properly for most cases.

**Comment by amcdavid** [comment]
Maybe ideal solution is to `setwd` to the project root before shelling out for any command, store relative paths, plus the project root, then `setwd` back to the original working directory upon function exit.
